src/components/welcome/MyAccounts/useAllSafes.ts

```diff
@@ -53,18 +51,16 @@ const useAllSafes = (): SafeItems | undefined => {
       const uniqueAddresses = uniq(addedOnChain.concat(ownedOnChain)).filter(Boolean)
 
       return uniqueAddresses.map((address) => {
-        const owners = allAdded?.[chainId]?.[address]?.owners
-        const isOwner = owners?.some(({ value }) => sameAddress(walletAddress, value))
         const isUndeployed = undeployedOnChain.includes(address)
-        const isOwned = (ownedOnChain || []).includes(address) || isOwner
+        const isOwned = (ownedOnChain || []).includes(address)
```

Comment:
Could you clarify this fix? It looks like it's now doing less than it did before.

Reply:
Due to this we get owned Safes from the added Safes while the reply from the owned Safes endpoint is still pending leading to us logging fewer owned Safes than we actually have.
E.g a user has 30 owned Safes but only 5 Safes added of which the connected wallet is also the owner.
We will return 5 owned Safes initially and then the 30 owned Safes leading to the tracking only capturing 5/30 Safes.

Reply:
But thinking about it some more this might affect counterfactual Safes as they are not part of the backend reply. Will double check that and def. write some tests for this hook. It has so much untested logic.